internal/generator/filters.go

```diff
@@ -3,27 +3,51 @@ package generator
 import (
 	"errors"
 	"fmt"
-	"strings"
 	"regexp"
+	"strings"
 
 	"github.com/flosch/pongo2/v5"
-	"github.com/labd/mach-composer/config"
+	"github.com/labd/mach-composer/internal/config"
 	"github.com/sirupsen/logrus"
 )
 
 func registerFilters() {
-	pongo2.RegisterFilter("string", filterString)
-	pongo2.RegisterFilter("short_prefix", filterShortPrefix)
-	pongo2.RegisterFilter("slugify", filterSlugify)
-	pongo2.RegisterFilter("remove", filterRemove)
-	pongo2.RegisterFilter("tf", filterTFValue)
-	pongo2.RegisterFilter("tfvalue", filterTFValue)
-	pongo2.RegisterFilter("render_tf_provider", renderTFProvider)
-	pongo2.RegisterFilter("azure_frontend_endpoint_name", AzureFrontendEndpointName)
-	pongo2.RegisterFilter("service_plan_resource_name", AzureServicePlanResourceName)
-	pongo2.RegisterFilter("get", FilterGetValueByKey)
-	pongo2.RegisterFilter("render_commercetools_scopes", filterCommercetoolsScopes)
-	pongo2.RegisterFilter("component_endpoint_name", filterComponentEndpointName)
+	if err := pongo2.RegisterFilter("string", filterString); err != nil {
+		panic(fmt.Errorf("pongo2.RegisterFilter('string'): %v", err))
+	}
```

Comment:
Perhaps we can better use `pongo2.Must(pongo2.RegisterFilter(...))`

Reply:
Is it `MustRegisterFilter`? Not sure, what this library does but `Must` seems to be something else.

Reply:
Added a `mustRegisterFilter()` function for this.